controller/dash_controller_server.py

```diff
@@ -0,0 +1,45 @@
+import socket
+import dash_circuit_board as dcb
+
+
+class DashControlServer:
+    """ DASH GC Controller - Triggers Actions """
+
+    def __init__(self, proc):
+        self.proc = proc
+        self.sock = None    # controller socket
+        self.d_sock = None  # detector socket
+        self.d_proc = None  # detector process
+        self.dcb_obj = dcb.DashCircuitBoard()
+
+    def configure_controller(self):
+        """ Configure the controller """
+
+        # create a UNIX Domain Socket
+        # (not available on windows so using a traditional socket)
```

Comment:
Add an `if` using `os` module to let UNIX users take advantage of UNIX domain sockets while letting Windows users get a functional traditional socket